infrastructure/networked_rds/prashant

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIJKQIBAAKCAgEAvX0fYXnUYJdQkwCXI0v6955Df356LdY0DSri/uSXzaE5476l
```

Comment:
this should not go into git. please do `git rm <file name>` and generate new keys for yourself

Reply:
yes